src/GenerateProgrammaticPipelineFromConfig/Generator.php

```diff
@@ -33,6 +33,7 @@ class Generator implements Constants
 use Zend\Expressive\Middleware\ImplicitHeadMiddleware;
 use Zend\Expressive\Middleware\ImplicitOptionsMiddleware;
```

Comment:
Change these to reference the zend-expressive-router variants, and also import their factories; the changes starting at line 60 then need updating.